examples/mobc/src/src_user/settings/modes/task_lists/elements/tl_elem_debug_display.c

```diff
@@ -8,13 +8,13 @@
 
 void BCL_load_tl_elem_debug_display_initial(void)
 {
-  BCL_tool_register_app    ( 0, AR_APP_DBG_FLUSH_SCREEN);
-  BCL_tool_register_app    ( 1, AR_APP_DBG_PRINT_GIT_REV);
-  BCL_tool_register_app    ( 2, AR_APP_DBG_PRINT_TIMESTAMP);
-  BCL_tool_register_app    ( 3, AR_APP_DBG_PRINT_CMD_STATUS);
-  BCL_tool_register_app    ( 4, AR_APP_DBG_PRINT_EVENT_LOGGER0);
-  BCL_tool_register_app    ( 5, AR_APP_DBG_PRINT_EVENT_LOGGER1);
-  BCL_tool_register_app    ( 6, AR_APP_DBG_PRINT_EVENT_HANDLER);
+  BCL_tool_register_app    ( 0, AR_APP_DBG_PRINT_GIT_REV);
+  BCL_tool_register_app    ( 1, AR_APP_DBG_PRINT_TIMESTAMP);
+  BCL_tool_register_app    ( 2, AR_APP_DBG_PRINT_CMD_STATUS);
+  BCL_tool_register_app    ( 3, AR_APP_DBG_PRINT_EVENT_LOGGER0);
+  BCL_tool_register_app    ( 4, AR_APP_DBG_PRINT_EVENT_LOGGER1);
+  BCL_tool_register_app    ( 5, AR_APP_DBG_PRINT_EVENT_HANDLER);
+  BCL_tool_register_app    ( 6, AR_APP_DBG_FLUSH_SCREEN);
   BCL_tool_register_app    ( 7, AR_NOP);
   BCL_tool_register_app    ( 8, AR_NOP);
   BCL_tool_register_app    ( 9, AR_NOP);
```

Comment:
```suggestion
  BCL_tool_register_app    ( 6, AR_NOP);
  BCL_tool_register_app    ( 7, AR_NOP);
  BCL_tool_register_app    ( 8, AR_NOP);
  BCL_tool_register_app    ( 9, AR_APP_DBG_FLUSH_SCREEN);
```

のほうがいいかもです（これだとスクリーンが30msぐらい消えてしまう？）

Reply:
表示は、消えなかったですが、TI＝９の方が、既存のタイミングにより近いですね。
修正しました。